client/my-sites/email/email-providers-comparison/index.jsx

```diff
@@ -309,6 +318,7 @@ class EmailProvidersComparison extends React.Component {
 
 	renderGoogleCard() {
 		const {
+			hasCartDomain,
```

Comment:
Nit: here and elsewhere, it is often easier to read if the props are in alphabetical order.

Reply:
Done